### ISSUE TITLE
Chromediver error breaking the documentation site accessibility scanner

### ISSUE THREAD

**Issue by timwright12** [department-of-veterans-affairs/va-mobile-app#10804]
There was a recent update to chromediver that is causing the accessibility scanner on the mobile documentation site to fail.

[Example failed run](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/runs/14792199115)

**Error**:

```
Error: session not created: This version of ChromeDriver only supports Chrome version 136
Current browser version is 135.0.7049.114 with binary path /opt/google/chrome/chrome
```

I tried to quickly patch this by pinning the chromedriver install to a past version (`npm install -g chromedriver@135.0.4`), but it didn't work.

**Acceptance Criteria**

- [ ] The daily accessibility scanner runs again

**Comment by jbergman-oddball** [comment]
It seems that ChromeDriver was bumped to version 136 2 days ago [here](https://github.com/giggio/node-chromedriver/commit/2ee268b0d0cfffeaae6d884be3b86497e1b0e916).  Going to update chrome to 136

**Comment by jbergman-oddball** [comment]
Update: We are installing version 136.  The image comes natively with Chrome 135 and ChromeDriver 135. [Reference](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#browsers-and-drivers)
Going to remove the 136 install and let it use the native packages.
